src/config/helpers.js

```diff
@@ -156,50 +156,29 @@ export function* getUniswapV2SethPrice(snxjs) {
 	);
 }
 
+export function* getCurveLatestSwapPrice(snxjs) {
+	const curveContract = new snxjs.ethers.Contract(
```

Comment:
hmm... this should probably be only instantiated once (out of the function)

Reply:
it only needs to be called once. We don't call the method again, so it is fine to do it here